sources/packages/web/src/views/student/financial-aid-application/ConfirmSubmission.vue

```diff
@@ -2,20 +2,27 @@
   <div class="p-component">
     <div class="p-card p-m-4">
       <Section title="Confirm Submission Page..." />
-      <FooterNavigator previous="financial-info" />
+      <FooterNavigator v-bind:previous="financialInfo" />
```

Comment:
Please condider use only the shortcut for v-bind (:previous="financialInfo") instead of (v-bind:previous="financialInfo").

Reply:
I just like using v-bind because it's more explicit. A little old school in this. Hope this is ok